real:
  parking_base_thresh: 0.1
  parking_base_speed: -0.2
  parking_base_yaw_target: 0.23 # times pi
  pixel_center_offset: -30
  constant_distance_to_intersection_at_detection: 0.3725
  change_lane_yaw: 0.29 # times pi
  change_lane_thresh: 0.05
  change_lane_speed: 0.2
  lane_relocalize: false
  sign_relocalize: false
  intersection_relocalize: false
  use_lane: false
  has_light: false
  change_lane_offset_scaler: 0.9
  sigma_delta: 10 # degrees
  sigma_v: 0.075
  cw_speed_ratio: 0.85
  hw_speed_ratio: 1.25
  sign_localization_threshold: 0.537
  lane_localization_threshold: 0.9
  lane_localization_orientation_threshold: 7.53 # degrees
  intersection_localization_threshold: 0.753
  intersection_localization_orientation_threshold: 15.0 # degrees
  odom_rate: 50
  stop_duration: 3.0
  use_stopline: true
  pedestrian_count_thresh: 8
  NORMAL_SPEED: 0.175
  FAST_SPEED: 0.4

sim:
  change_lane_yaw: 0.25 # times pi
  change_lane_speed: 0.2
  change_lane_thresh: 0.05
  cw_speed_ratio: 0.7143
  hw_speed_ratio: 1.33
  sign_localization_threshold: 0.537
  lane_localization_threshold: 0.1
  lane_localization_orientation_threshold: 7.53 # degrees
  intersection_localization_threshold: 1.5
  intersection_localization_orientation_threshold: 15.0 # degrees
  constant_distance_to_intersection_at_detection: 0.3725
  pixel_center_offset: -30
  sigma_v: 0.075
  sigma_delta: 10 # degrees
  odom_rate: 50
  stop_duration: 1.5
  use_stopline: true
  pedestrian_count_thresh: 8
  parking_base_yaw_target: 0.29 # times pi
  parking_base_speed: -0.2
  parking_base_thresh: 0.1
  NORMAL_SPEED: 0.23
  FAST_SPEED: 0.4
  lane_relocalize: false
  sign_relocalize: true
  intersection_relocalize: false
  use_lane: false
  has_light: false
  change_lane_offset_scaler: 1.3